index.js

```diff
@@ -165,47 +165,29 @@ function loadConfigFile (configPath) {
 
     // ES6 export default
     if (config && config.default) {
-      return config.default
+      config = config.default
     }
 
-    return config
-  } catch (e) {
-    // Do nothing
-  }
-}
-
-// Load configuration from the package.json file
-function loadPackageJSONFile (configPath) {
-  try {
-    var config = require(configPath)
-
     if (config && config['module-alias']) {
```

Comment:
Maybe in this case, do an "else" ; because first case is "es6 imports" and second case is "package.json". It is functionally very similar but conveys more intent.

Reply:
What about?

op#1
```
    if (!config) {
      return null
    } else if (config.default) { // ES6 export default
      config = config.default
    } else if (config['module-alias']) {
      return config['module-alias']
    } else {
      return config
    }
```
The first `!config` means that we don't have to check for `config` twice if it has value.

Another alternative could be
op#2
```
// ES6 export default
    if (config && config.default) {
      config = config.default
    } else if (config && config['module-alias']) {
      return config['module-alias']
    } else {
      return config
    }
```

OR

op#3
```
 // ES6 export default
    if (config && config.default) {
      config = config.default
    } else if (config && config['module-alias']) {
      config = config['module-alias']
    }

    return config
```

Reply:
@Kehrlann